test/ffmpeg-qsv/encode/avc.py

```diff
@@ -82,13 +83,13 @@ def init(self, tspec, case, gop, slices, qp, quality, profile):
     )
 
   @slash.parametrize(*gen_avc_cqp_lp_parameters(spec, ['high', 'main']))
-  def test(self, case, gop, slices, qp, quality, profile):
```

Comment:
@uartie I understand your concern. Yes, It will lost history data. Actually I also think about this before. 
One thing need to let you do compare, because existing test cases only do test for I, IP frame mode for LP AVC CQP/CBR/VBR with high/main profile and a lot of resolution, slices (...ect, parameter), From user side, IPB mode is more usage than I/IP mode, so IPB will also need to cover these testing parameter.
That means if to submit new test cases for test_bframes, there will have another new test functions name of gen_avc_cqp_lp_bframes_parameters/gen_avc_cbr_lp_bframes_parameters/gen_avc_vbr_lp_bframes_parameters..., and will have new class with names of cqp_lp_bframes, cbr_lp_bframes, vbr_lp_bframes almost same test parameters with cqp_lp/cbr_lp/vbr_lp in each ffmpeg/gstramer encode/avc.py files. This is why I submitted vaapi-fits patches with all full/sanity/bat AVC LP encode test cases reference rebased. Currently, existing AVC LP encode test case results are good, maybe we can use this chance to refined the AVC LP test reference.
How do you think?